cmake/Platform/Linux/Packaging_linux.cmake

```diff
@@ -25,26 +25,27 @@ elseif("$ENV{O3DE_PACKAGE_TYPE}" STREQUAL "DEB")
 
     set(CPACK_PACKAGING_INSTALL_PREFIX "/opt/${CPACK_PACKAGE_NAME}/${CPACK_PACKAGE_VERSION}")
 
-    # get all the package dependencies, extracted from scripts\build\build_node\Platform\Linux\package-list.ubuntu-focal.txt
+    # Define all the debian package dependencies needed to build and run
     set(package_dependencies
-        libffi7
-        clang-12
+        # Required Tools
+        "cmake (>=3.22)"                        # Cmake required (minimum version 3.22.0)
+        "clang (>=12.0)"                        # Clang required (minimum version 12.0)
```

Comment:
Nice. Something we should have thought to do earlier

Reply:
I didn't know apt-get supported dependency strings.
Apparently this was added in 2019 as part of the [apt-get satisfy](https://askubuntu.com/questions/1413251/how-does-the-apt-satisfy-work) command

Reply:
https://cmake.org/cmake/help/v3.23/cpack_gen/deb.html?highlight=cpack_debian_package_depends#variable:CPACK_DEBIAN_PACKAGE_DEPENDS

Reply:
Yeah, though dep strings have actually been a feature for CMake since 3.3 (as @spham-amzn just linked)

Reply:
So CPack supports it as well.

That gives me an idea that we should update the Linux environment setup scripts to use `apt-get satisfy` then for